src/NServiceBus.AcceptanceTests/Core/Recoverability/When_configuring_unrecoverable_exception.cs

```diff
@@ -13,6 +13,8 @@ public class When_configuring_unrecoverable_exception : NServiceBusAcceptanceTes
         [Test]
         public void Should_move_to_error_queue_without_retries()
         {
+            Requires.DelayedDelivery();
```

Comment:
I think we might be able to skip the delayed retries in the custom policy and remove this ?

Reply:
The reason I kept this one is that if delayed retries are disabled, are we really sure that the unrecoverable exception is working properly, or did it just go to the error queue because they were disabled? In theory, it could delay the message and then the 3rd assertion in the test would fail.